src/router/routes.ts

```diff
@@ -15,14 +15,22 @@ const routes: readonly RouteRecordRaw[] = [
     name: 'user-management',
     component: () => import('../views/admin/UserManagementView.vue'),
     meta: {
-      layout: 'AdminLayout',
-      requiresAuth: true
+      requiresAuth: true,
+      layout: 'AdminLayout'
     }
   },
   {
     path: '/admin/users/:id',
     name: 'user-details',
     component: () => import('../views/admin/UserDetailsView.vue'),
+    meta: {
+      requiresAuth: true
```

Comment:
```suggestion
      requiresAuth: true,
      layout: 'AdminLayout'
```
Sonst wird das Menü nicht angezeigt